build.sbt

```diff
@@ -131,17 +131,19 @@ lazy val commonSettings = Seq(
   fork in Test := true,
   resolvers += Resolver.mavenLocal,
   concurrentRestrictions in Global += Tags.limit(Tags.Test, 1),
+  scalaVersion := "2.13.0",
+  crossScalaVersions := Seq("2.10.7", "2.11.12", "2.12.7", "2.13.0"),
```

Comment:
Could you please remove Scala 2.10? We are no longer supporting Scala 2.10 since Kamon 2.0. By the way, if you are interested in having this change on the 1.x series it would be awesome to get a PR on the `maintenance-v1.1` branch :smile:

Reply:
Commit added. May find tuits for t'other PR, but no promises... Thanks for the very prompt review!